exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/handlers/SetOptionHandler.java

```diff
@@ -63,9 +64,10 @@ public final PhysicalPlan getPlan(SqlNode sqlNode) throws ForemanSetupException
     SqlNode optionValue = statement.getValue();
 
     if (optionValue == null) {
+      OptionDefinition optionDefinition = optionManager.getOptionDefinition(optionName);
       String value = String.valueOf(optionManager.getOption(optionName).getValue());
 
-      return DirectPlan.createDirectPlan(context, new SetOptionViewResult(optionName, value));
+      return DirectPlan.createDirectPlan(context, new SetOptionViewResult(optionDefinition.getValidator().getOptionName(), value));
```

Comment:
The naive reader (that would be me) would wonder why not use the `optionName` rather than using this indirect approach. Maybe a brief comment saying that we're doing it this way to use the name as defined in the option, rather than what the user provided.

Reply:
Thanks, added a comment with clarification.